Marlin/src/gcode/calibrate/M48.cpp

```diff
@@ -246,6 +248,11 @@ void GcodeSuite::M48() {
 
     SERIAL_ECHOLNPAIR_F("Standard Deviation: ", sigma, 6);
     SERIAL_EOL();
+
+    #if HAS_SPI_LCD
+      // Display M48 results in the status bar
+      ui.status_printf_P(0, PSTR(MSG_M48_DEVIATION ": %.6f"), sigma);
```

Comment:
Fun fact: You can't use "`%f`" in printf-like functions on Arduino. It isn't implemented.